recipes/ta-numba/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: ta-numba
+  version: "0.2.2"
+
+source:
+  url: https://pypi.io/packages/source/t/ta-numba/ta_numba-0.2.2.tar.gz
+  sha256: f3b06db354ed0e31651590e433a88f6f163ec6ff3fda12380f464bf100e5604c
+
+build:
```

Comment:
you need a build `number`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L39